### ISSUE TITLE
Need proper ways to handle `aria-attributes`

### ISSUE THREAD

**Issue by Gr8Gatsby** [salesforce/lwc#224]
_SOMA#825_

### Migrated issue

notes:

The subject actually doesn't really describe the whole picture.

A few issues here,
- in pure raptor context, `aria-attributes` can be set to any custom elements and they will be set to the container. When raptor components are used in aura, the `aria-attributes` don't work:
`org.auraframework.throwable.quickfix.AttributeNotFoundException: The attribute "aria-describedby" was not found on the COMPONENT markup://lightning:input`
- adding `@api ariaDescribedby` for example allows this to work:
```html
<lightning:input name='input' ariaDescribedby='abc' />
```
This doesn't affect how `aria-describedby` behaves in pure raptor context. Seems like this is a bug in the interop layer and should not be allowed based on @cpatino 
- need a way to set `aria-attributes` on internal elements. For example, the a11y team said users should be able to add more `aria-describedby` to `<input>`. I'm not very familiar with a11y stuff, so I'm not sure if just setting it on the raptor container element would work.
